pkgs/development/haskell-modules/patches/hledger-properly-escape-quotes-csv.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/hledger-lib/Hledger/Read/CsvReader.hs b/hledger-lib/Hledger/Read/CsvReader.hs
```

Comment:
This patch only includes the changed `CsvReader.hs` but no updated tests. How is this not an issue? You could also just `fetchpatch` the commit.

Reply:
+1 for using fetchpatch (just to reduce the size of nixpkgs :))

Reply:
> This patch only includes the changed CsvReader.hs but no updated tests. How is this not an issue?

The [original patch](https://github.com/simonmichael/hledger/pull/1512) doesn't modify any test, it just adds one. Unfortunately, the test being added is for another part of the repository which is in another nixpkgs' package: said differently, only a test for the application was added, not a unit test of the library.

> You could also just fetchpatch the commit.

good idea, done.